src/content/get-started/fundamentals/dart.md

```diff
@@ -48,7 +48,7 @@ class Package {
   final String latestVersion; 
   final String? description;
 
-  Package(this.name, this.latestVersion, this.description);
+  Package(this.name, this.latestVersion, {this.description});
```

Comment:
I don't think this is the correct solution. Shouldn't an optional named parameter be put in square brackets?

@ericwindmill

Reply:
This is correct, named parameters use curly braces